packages/file-search/src/node/file-search-service-impl.ts

```diff
@@ -141,7 +141,7 @@ export class FileSearchServiceImpl implements FileSearchService {
     }
 
     private getSearchArgs(options: FileSearchService.BaseOptions): string[] {
-        const args = ['--files', '--case-sensitive'];
+        const args = ['--files', '--hidden', '--case-sensitive']
```

Comment:
it should not affect all clients, but only a command.

Reply:
Although I'm not sure, maybe it is fine to have it always on.

cc @eclipse-theia/ecd-theia-committers ?

Reply:
> it should not affect all clients, but only `vscode.workspace.findFiles` command, i.e. with a new option.

But It's not for Find In Files command, it's for vscode API vscode.workspace.findFiles function.

Reply:
> > it should not affect all clients, but only `vscode.workspace.findFiles` command, i.e. with a new option.
> 
> But It's not for Find In Files command, it's for vscode API vscode.workspace.findFiles function.

It likely affects both, perhaps it is fine to keep it on as Anton mentioned.